common/djangoapps/django_sudo/decorators.py

```diff
@@ -0,0 +1,55 @@
+"""
```

Comment:
High level question - why is this package named `django_sudo`? I'm finding this confusing, since the library itself is called `django-sudo`. Would it be alright to call this something along the lines of `django_sudo_helpers`?